portafly/portafly_api.yml

```diff
@@ -36,95 +36,234 @@ paths:
       schema:
         type: string
       in: query
+  /applications:
+    summary: A list of all applications
+    description: A list of all applications
+    get:
+      responses:
+        "200":
```

Comment:
We are using single quotes in the other responses:
https://github.com/3scale/porta/blob/master/portafly/portafly_api.yml#L17